### PR TITLE
add p9dsu project.

### PR DESCRIPTION
**Pull request by jimsmc**
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/1009)

### CODE REVIEW

**Comment by wghoffa** [comment]
LGTM

---

Review status: 0 of 5 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/open-power/op-build/1009#-:-KhP4HijRGxnBQVSXqx1:bb6ywmx)*

**Comment by wghoffa** [comment]
Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/open-power/op-build/1009)*

File: openpower/configs/hostboot/p9dsu.config [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# The Serial Flash Controller is the AST2400 BMC.
```
Comment (shenki) [inline openpower/configs/hostboot/p9dsu.config]: AST2500

File: openpower/configs/p9dsu_defconfig [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+BR2_HOSTBOOT_BINARY_WINK_FILENAME="p9n.ref_image.hdr.bin.ecc"
+BR2_IMA_CATALOG_FILENAME="ima_catalog.bin"
+BR2_OPENPOWER_TARGETING_BIN_FILENAME="P9DSU_HB.targeting.bin"
+BR2_OPENPOWER_TARGETING_ECC_FILENAME="P9DSU_HB.targeting.bin.ecc"
+BR2_PACKAGE_PETITBOOT=y
+BR2_PACKAGE_PETITBOOT_MTD=y
+BR2_OCC_BIN_FILENAME="occ.bin"
+BR2_CAPP_UCODE_BIN_FILENAME="cappucode.bin"
+BR2_SKIBOOT_CUSTOM_VERSION=y
+BR2_SKIBOOT_CUSTOM_VERSION_VALUE="skiboot-5.5.0-rc2"
```
Comment (shenki) [inline openpower/configs/p9dsu_defconfig]: Please rebase your commit on the latest master. Amongst other things, this has changed.

**Comment by mikey** [comment]
I'd prefer if we wait on this until the skiboot bit has been merged.

**Comment by stewart-ibm** [comment]
I rebased on master, fixed up the skiboot verison and pushed to master as of aa0c5431f06f

**Pull request closed without merging**